.babelrc.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
     [
       "@4c/4catalyzer",
       {
-        target: "node"
+        target: "node",
+        include: [],
```

Comment:
Do you need the include?

Reply:
yes, because I don't want [this](https://github.com/4Catalyzer/javascript/blob/afe962861fd63f0812a86d5c7cf1859370deaec3/packages/babel-preset/index.js#L44). the problem is that if this is transformed to a non-class but the super class is still a class, then it's not correct. Maybe there are other ways to circumvent this, not sure. ideally that include should only be on the web preset, not the node, I guess

Reply:
ah ok, yes we should only include for `web`

Reply:
https://github.com/4Catalyzer/javascript/pull/47

Reply:
🙏